src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -33,6 +33,8 @@ public class ProcessedArgs
         private readonly string sonarQubeUrl;
         private readonly IAnalysisPropertyProvider globalFileProperties;
 
+        protected /* for testing */ ProcessedArgs() { }
```

Comment:
It is indeed not referenced anywhere in the codebase, but it is needed for `Moq` to do it's magic. Otherwise something like this fails:

```csharp
   var mock = new Mock<ProcessedArgs>();
   var actual = mock.Object // <- this throws 
```

(We use this functionality on the SonarCloud Tests)